transform_data.py

```diff
@@ -31,15 +39,17 @@ def add_stock_returns(stock_history:pd.DataFrame) -> pd.DataFrame:
         pd.DataFrame: stock history with 2 return added
     """
     try:
-        stock_history.sort_values(by = 'Date', ascending= True, inplace = True)
-        stock_history['daily_return'] = stock_history['Close'].pct_change()
-        stock_history['cummulative_return'] = (1 + stock_history['daily_return']).cumprod() - 1
```

Comment:
magical number: https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad
ZERO = 0
ONE = 1